components/director/internal/graphql/scalars/helpers.go

```diff
@@ -0,0 +1,52 @@
+package scalars
+
+import (
+	"encoding/json"
+	"io"
+
+	"github.com/pkg/errors"
+)
+
+func marshalAndWrite(in interface{}, w io.Writer) error {
```

Comment:
```suggestion
func marshallToWriter(in interface{}, w io.Writer) error {
```

Reply:
Hmm, marshallToWriter is confusing. I had two other ideas: 
`writeMarshalled` or `writeBytes`

Reply:
changed to `writeMarshalled`